From 6b0efe545337fe6412a012d025f51ec06e524c15 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 29 Nov 2024 14:41:12 +0100
Subject: [PATCH] meson: Drop max version bound from neon

Neon 0.34.0 broke the build again, but the API+ABI has been stable since
0.27 and the library is so-versioned.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/8008>
---
 subprojects/gst-plugins-bad/ext/neon/meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ext/neon/meson.build b/subprojects/gst-plugins-bad/ext/neon/meson.build
index 7aa5ed78d5c9..7c2696cfa106 100644
--- a/ext/neon/meson.build
+++ b/ext/neon/meson.build
@@ -1,5 +1,4 @@
-neon_dep = dependency('neon', version: ['>= 0.27', '<= 0.33.99'],
-                      required : get_option('neon'))
+neon_dep = dependency('neon', version: '>= 0.27', required: get_option('neon'))

 if neon_dep.found()
   gstneon = library('gstneonhttpsrc',
--
GitLab

